typings/src/lib/generators.d.ts

```diff
@@ -0,0 +1,180 @@
+      * @param {string} options.image - The link to an image (.png, .jpg, .gif)
+      * @param {string} options.title - The title text to print.
+      * @param {string} options.text - The text to print.
+      * @returns {Promise<buffer>} The generated image in a buffer.
+      */
+    demotivational(options?: {
+        image: string;
+        title: string;
+        text: string;
+    }): Promise<any>;
+    /**
+      * Draws a image, username and message on a Gravestone. Please see https://weebyapi.xyz/api/docs#generators for the available generators.
+      * @param {Object} options - The options that contain the required parameters.
+      * @param {string} options.avatar - The link to an image (.png, .jpg, .gif)
+      * @param {string} options.username - The username to print.
+      * @param {string} options.message - The text to print.
+      * @returns {Promise<buffer>} The generated image in a buffer.
+      */
+    rip(options?: {
+        avatar: string;
+        username: string;
+        message: string;
+    }): Promise<any>;
+    /**
+      * Generates a raw twitter tweet with username, tweet text and avatar. Please see https://weebyapi.xyz/api/docs#generators for the available generators.
+      * @param {Object} options - The options that contain the required parameters.
+      * @param {string} options.username - The username to print.
+      * @param {string} options.tweet - The text to print.
+      * @param {string} options.avatar - The link to an image (.png, .jpg, .gif)
+      * @returns {Promise<buffer>} The generated image in a buffer.
+      */
+    tweet(options?: {
+        username: string;
+        tweet: string;
+        avatar: string;
+    }): Promise<any>;
+    /**
+      * Generates a twitter tweet with username (fetched from the Twitter API) and the tweet text. Please see https://weebyapi.xyz/api/docs#generators for the available generators.
+      * @param {Object} options - The options that contain the required parameters.
+      * @param {string} options.username - The username to print.
+      * @param {string} options.tweet - The text to print.
+      * @returns {Promise<buffer>} The generated image in a buffer.
+      */
+    tweetFetch(options?: {
+        username: string;
+        tweet: string;
+    }): Promise<any>;
+    /**
+      * Generates a Spotify Banner of statistics and information from either music, playlists or users. Please see https://weebyapi.xyz/api/docs#generators for the available generators.
+      * @param {Object} options - The options that contain the required parameters.
+      * @param {('album'|'artist'|'playlist'|'track'|'user')} options.type - The type of Spotify Search. (album, artist, playlist, track or user)
+      * @param {string} options.query - The value to search up. (Use %20 for spaces)
+      * @returns {Promise<buffer>} The generated image in a buffer.
+      */
+    spotifySearch(options?: {
```

Comment:
Spotify Search Endpoint was permanently removed due to issues with spotify api

Reply:
This still exists in the JS code?
https://github.com/ntm-development/weeby-js/blob/ee8202613c2a1b287594c8dbee4e6ec43e78a8d1/src/lib/generators.js#L314-L341

Reply:
Should this be removed too?

Reply:
The next update for Weeby-JS will have features added from the recent Weeby API update. So yes, Spotify Search will not be needed anymore